guides/advanced_guides/configuration.md

```diff
@@ -69,7 +69,21 @@ The master key allowing you to do everything on the server. If no master key is
 **Environment variable**: `MEILI_NO_ANALYTICS`
 **CLI option**: `--no-analytics`
 
-Deactivates analytics. Analytics allow us to know how many users are using MeiliSearch, which versions and which platforms are used. This process is entirely anonymous.
+Deactivates analytics.
+
+Analytics allow us to know how many users are using MeiliSearch and the following:
+
+- The application version.
+- The environment: development/production.
+- The number of days since the first start: Segment development/production.
+- The user email: (*Optional*) If the user wants to have alerts.
+- The server provider: (*Optional*).
+- The database size.
+- The last update.
+- The number of updates per index.
+- The number of documents per index.
+
+**The index names are hashed.**
```

Comment:
We didn't retrieve the index names at all.

Reply:
```suggestion
```